l4ping/l4ping.TESTGEN.ps1

```diff
@@ -0,0 +1,143 @@
+#===============================================
+# Script Input Parameters Enforcement
+#===============================================
+Param(
+    [parameter(Mandatory=$false)] [string] $Config = "Default",
+    [parameter(Mandatory=$true)]  [string] $DestIp,
+    [parameter(Mandatory=$true)]  [string] $SrcIp,
```

Comment:
Parameter validation to make sure the string is a valid IP address.

Reply:
Added it.